pythagorean_thm.py

```diff
@@ -0,0 +1,27 @@
+'''
+This script will help solve the pythagorean theorem.
+'''
+
+import math
+
+def main(a, b, c):
+    ''' Solve the the pythagorean theorem given inputs. '''
+
+    if c is None:
+        c = math.sqrt(pow(a, 2) + pow(b, 2))
+        print(f'The answer is {c}')
+
+    if a is None:
+        a = math.sqrt(pow(c, 2) - pow(b, 2))
```

Comment:
The logic of this line is very similar to line 19. Consider creating a function to call in both places instead.